conformance/README.md

```diff
@@ -3,8 +3,53 @@
 This directory contains conformance tests for Ion, expressed using a small,
 declarative domain-specific language.
 
+**Contents**
+ * `core/` – test cases for verifying the functionality of a conformance DSL runner
+ * `eexp/` – test cases for parsing/interpretation of e-expressions and their arguments
```

Comment:
What I'm trying to say is that this test for e-expression syntax/encoding without trying to specifically verify the implementation details of any specific macro. So, it would cover whether the arguments are correctly matched to parameters, that the opcodes, `(:`, `(::`, presence bitmaps, etc. are handled correctly.

Reply:
Makes sense. In that case I don't have a better phrase than "interpretation."